Src/Support/Google.Apis.Auth/OAuth2/LocalServerCodeReceiver.cs

```diff
@@ -65,6 +67,69 @@ public class LocalServerCodeReceiver : ICodeReceiver
   </body>
 </html>";
 
+        private static object s_lock = new object();
+        // Current best-guess as to most suitable callback URI.
+        private static string s_callbackUriTemplate;
+        // Has a successful callback been received?
+        private static bool s_receivedCallback;
+
+        /// <summary>
+        /// Create an instance of <see cref="LocalServerCodeReceiver"/>.
+        /// </summary>
+        public LocalServerCodeReceiver()
+        {
+            lock (s_lock)
+            {
+                // Listening on 127.0.0.1 is recommended, but can't be done in non-admin Windows 7 & 8.
+                // So use some tests/heuristics so maybe listen on localhost instead.
+                if (s_callbackUriTemplate != null && !s_receivedCallback)
+                {
+                    // On non-first runs, if a successful callback has not been received
+                    // then force callback to use localhost rather than 127.0.0.1
+                    // This is to heuristically avoid errors on browsers that can't connect to 127.0.0.1
+                    // E.g. IE11 in enhanced protection mode.
+                    s_callbackUriTemplate = CallbackUriTemplateLocalhost;
```

Comment:
No, it'll work fine.
This is purely so the browser uses "localhost" rather than "127.0.0.1". As far as the netstandard server is concerned they are identical. It just listens on 127.0.0.1, and doesn't care how the browser contacted it.
The Net45 server is different, and requires the client and server to match on the exact uri.